docs/entanglement_forging/how-tos/freeze-orbitals.ipynb

```diff
@@ -71,13 +71,43 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Create the ansatz with a reduced set of bitstrings."
+    "Since the execution time scales exponentially in the number of orbitals, it’s a good idea to simplify the problem (if possible) by eliminating some of the orbitals. Some knowledge of chemistry is useful when picking orbitals to freeze. One good rule of thumb is to freeze the core orbital (for the case of water, this is the core oxygen 1s orbital). Furthermore, in the case of water, it turns out that orbital 3 (corresponding to the out-of-plane oxygen 2p orbitals) has different symmetry to the other orbitals, so excitations to orbitals 1 and 3 are suppressed. For water, we thus freeze orbitals 0 and 3, resulting in length-5 bitstrings."
```

Comment:
> so excitations to orbitals 1 and 3 are suppressed. For water, we thus freeze orbitals 0 and 3

How do you get from "1 and 3" to "0 and 3"?  It might be obvious if I thought a little bit, but I think it'd be useful to be more explicit about this.

Reply:
Yes, I mostly copy-pasted this. I think what they were getting at is that the 1s oxygen orbital is the core orbital of water, and that orbital is referenced through bitstring-index 0. The 2p oxygen orbitals have this described asymmetry, and they are referenced via bitstring-index 3.

You do need chemistry knowledge to start manipulating these things, but I will clear up this statement.